src/test/java/com/connexta/store/adaptors/S3StorageAdaptorITests.java

```diff
@@ -6,23 +6,28 @@
  */
 package com.connexta.store.adaptors;
 
+import static com.connexta.store.adaptors.StoreStatus.STAGED;
+import static com.connexta.store.adaptors.StoreStatus.STATUS_KEY;
+import static com.connexta.store.adaptors.StoreStatus.STORED;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
✏️ I think `assertThat` is preferred over `assertEquals` and `assertTrue` because it has better failure messages.

Reply:
I'll update them.